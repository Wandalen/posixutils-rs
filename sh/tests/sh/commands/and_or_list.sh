true && echo correct
echo $?
false && echo wrong
echo $?
true || echo wrong
echo $?
false || echo correct
echo $?
true && echo correct || echo wrong
echo $?
false && echo wrong || echo correct
echo $?

