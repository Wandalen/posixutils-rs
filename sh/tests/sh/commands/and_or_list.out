correct
0
1
0
correct
0
correct
0
correct
0
